plugins/source/k8s/gen.sh

```diff
@@ -0,0 +1,6 @@
+mockgen --package=mocks --destination="client/mocks/mock.go" "k8s.io/client-go/kubernetes" Interface
```

Comment:
Maybe we can put this in the Makefile under:
https://github.com/cloudquery/cloudquery/blob/43d5fcd77160c4803f88ddce55cd96969da66e21/plugins/source/k8s/Makefile#L25

or `gen-mocks:` and update `gen:`?